pkg/apiclient/apiclient.go

```diff
@@ -171,6 +180,9 @@ func NewClient(opts *ClientOptions) (Client, error) {
 	if opts.Insecure {
 		c.Insecure = true
 	}
+	if opts.GRPCWeb {
```

Comment:
So just like we have an Env variable for ARGOCD_SERVER, I think we may need env var for ARGOCD_GRPCWEB. Maybe we can generalize this to ARGOCD_OPTS to support something like:
```
ARGOCD_SERVER=argocd.example.com
ARGOCD_TOKEN=<some jwt token>
ARGOCD_OPTS="--grpc-web --insecure"

argocd app sync guestbook
```

Reply:
I'm okay addressing this later, so feel free to file a new issue for ARGOCD_OPTS .

Reply:
I like ARGOCD_OPTS proposal. Creating an issue for it.